modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties

```
@@ -807,7 +807,7 @@ key-help=Provide a unique identifier that can be used to refer to it programmati
 key-version=Key Version
 kpis=KPIs
 last-day-of-month=Last Day of Month
-last-publish-date=Last publish date
+last-publish-date=Last publish Date
```

@andrea-ale-sbarra capitalize like this since its a title.
```suggestion
last-publish-date=Last Publish Date
```